src/org/omegat/core/team2/impl/HTTPRemoteRepository.java

```diff
@@ -217,7 +217,8 @@ protected void retrieve(Properties etags, String file, String url, final File ou
                 logger.atDebug().setMessage("Retrieve {0}: not modified").addArgument(url).log();
                 return;
             default:
```

Comment:
It is better to handle more precise detection of the reasons.
We can use different message for 404 and 431 error.

Reply:
I want to check forbidden, unauthorized and not_found errors.